app/models/organisation.rb

```diff
@@ -58,6 +58,7 @@ def affiliated_stock_owners
   alias_method :la?, :LA?
 
   validates :name, presence: { message: I18n.t("validations.organisation.name_missing") }
+  validates :name, uniqueness: { case_sensitive: false, message: I18n.t("validations.organisation.name_not_unique") }
```

Comment:
Should we be able to use a name that had been used by an organisation that is now deleted?

I suspect the answer is no, but that might be a bit weird if users get this message but can't find an org with that name.

Reply:
I see what you mean, from what I understand we don't have duplicates atm anyway and if we ever have a requirement that an org can be reinstated it might cause issues if we don't check deleted org names.

I don't think this would be a common scenario. Let me know if you disagree tho